### DISCUSSION TITLE
ERR_SSL_PROTOCOL_ERROR на сайтах dns, авито, хабр

### DISCUSSION THREAD

**Discussion by brnsky** [bol-van/zapret#986] [Q&A] [answered]
При включенном zapret заблокированные сайты обходятся. Если включаю впн с zapret, то все начинает дохнуть. Также появляется ошибка при подключении к некоторым сайтам при включенном zapret. Через исключения не работает фикс. Тогда все перестает блочиться. Система - мак ос
[config — копия.txt](https://github.com/user-attachments/files/18237943/config.txt)

**Comment by bol-van** [comment]
Неработа стратегии - это не issue.

Это известно, что стратегии могут ломать сайты, и нет однокнопочного решения
На другом подключении может вообще все сломать, если гнать чужую стратегию

**Reply by brnsky** [reply]
Спасибо, но вот я добавил в исключения и поставил параметры подходящие, но ВПН также дохнет и не подключается, в самом Впн выбирают туннелирование только одного приложения (дискорда). Но он не подключается, без включенного запрет - работает

**Reply by brnsky** [reply]
Как проблему пофиксить с впн - не знаю, но там также исключения не помогают

**Reply by bol-van** [reply, marked as answer]
Можно IFACE_WAN заполнить или выбрать соотв пункт в инсталяторе.
Вписать туда имена физических wifi и ethernet интерфейсов через пробел в кавычках
Тогда исходящее с этой системы на другие интерфейсы не будет направляться на tpws

Но если само VPN подключение идет через tls на 443 порту, то его тоже может сломать
надо --debug лог будет смотреть  на что оно реагирует и вносить исключения для VPN сервера

**Reply by brnsky** [reply]
> Можно IFACE_WAN заполнить или выбрать соотв пункт в инсталяторе.
> Вписать туда имена физических wifi и ethernet интерфейсов через пробел в кавычках
> Тогда исходящее с этой системы на другие интерфейсы не будет направляться на tpws
> 
> Но если само VPN подключение идет через tls на 443 порту, то его тоже может сломать
> надо --debug лог будет смотреть  на что оно реагирует и вносить исключения для VPN сервера

Удивительно, сегодня утром открыл компьютер (на макос) и ничего кроме запрета включено не было (ВПН, прокси и ТД) и Дискорд заработал хотя он в исключениях прописан, настройки и ничего не менял. Войсы заработали. Не знаю с чем связано это, а теперь не работают

**Reply by brnsky** [reply]
Причем zapret я не вырубал никоим образом, но заработало идеально без пинга

**Reply by bol-van** [reply]
Войсы в дискорде работают по udp. tpws удп не поддерживает.
Наверно временно отключили блокиратор
